src/contracts/pods/EigenPod.sol

```diff
@@ -172,41 +172,31 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         emit NonBeaconChainETHReceived(msg.value);
     }
 
+//     BeaconState(epochToTimestamp(validator.withdrawableEpoch))
+//     .next_withdrawal_validator_index 
+// <= validator.Index 
+// < BeaconState(oracleTimestamp).next_withdrawal_validator_index
+
```

Comment:
why is this comment being added here? seems totally out of place

Reply:
whoops that was supposed to be deleted, mbad